cmd/otelcontribcol/builder-config.yaml

```diff
@@ -416,3 +416,6 @@ replaces:
   - github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector => ../../connector/spanmetricsconnector
   - github.com/openshift/api v3.9.0+incompatible => github.com/openshift/api v0.0.0-20180801171038-322a19404e37
   - github.com/outcaste-io/ristretto v0.2.0 => github.com/outcaste-io/ristretto v0.2.1
+  - github.com/DataDog/datadog-agent/pkg/proto => github.com/DataDog/datadog-agent/pkg/proto v0.48.0-beta.1
+  - github.com/DataDog/datadog-agent/pkg/trace => github.com/DataDog/datadog-agent/pkg/trace v0.48.0-beta.1
```

Comment:
these replace statements are odd to me, why are they present?

Reply:
I'm not entirely sure. Somewhere along the way the got included, they may have been included in another package.  I can remove them. I've been more focused on fixing the build. I can look at removing that though.